docs/user/ppl/general/identifiers.rst

```diff
@@ -83,6 +83,25 @@ Here are examples for quoting an index name by back ticks::
     +------------------+
 
 
+Cross-Cluster Index Identifiers
+===================
+
+Description
+-----------
+
+A cross-cluster index identifier is an index identifier with a prefix ``<cluster identifier>:``. The cluster identifier could contain star ``*``. This is mostly an cluster pattern for wildcard match.
```

Comment:
Similar question to the above - why use a different character `:` for a similar meaning as in datasource ?